src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+            </tbody>
+        </table>
+        <h4>Type 13 Protobuf Data</h4>
+        <p>Supported from Add-On protocol version 1.2.</p>
+        <p>This message allows an IOX to send a protobuf-encoded payload to the GO device. It supports a publish/subscribe model of vehicle status information. The GO device responds with GO Multi-Frame Data (0x27) - Type 13. <a href="https://github.com/Geotab/android-external-device-example/blob/master/app/src/main/proto/iox_messaging.proto" target="_blank" rel="noreferrer">Protobuf Schema</a>. The currently supported topics are:</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Topic</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>TOPIC_VIN</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_GEAR</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_ENGINE_SPEED</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_ENGINE_LOAD</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_ODOMETER</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_ACCEL_PEDAL_PERCENTAGE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_COOLANT_TEMP</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_DOC_INTAKE_GAS_TEMP</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_DOC_OUTLET_GAS_TEMP</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK1_UNITS</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK2_UNITS</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK1_PERCENT</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK2_PERCENT</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_STATE_OF_CHARGE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_ENGINE_ROAD_SPEED</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_VEHICLE_ACTIVE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_DRIVER_SEATBELT</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_LEFT_TURN_SIGNAL</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_RIGHT_TURN_SIGNAL</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_EV_CHARGING_STATE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_PARK_BRAKE</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Buzzer Beep (0x24)</h3>
+        <p>Sent from an IOX to the GO device to request the buzzer beep with the given parameters.</p>
+        <h4>Payload</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on